lib/Doctrine/Inflector/Rules/Dutch/Inflectible.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Inflector\Rules\Dutch;
+
+use Doctrine\Inflector\Rules\Pattern;
+use Doctrine\Inflector\Rules\Substitution;
+use Doctrine\Inflector\Rules\Transformation;
+use Doctrine\Inflector\Rules\Word;
+
+/**
+ *  // http://nl.wikipedia.org/wiki/Klinker_(klank)
+ *  private $klinker = '(a|e|i|o|u|ij)';
+ *  private $korteKlinker = '(u|i|e|a|o)';          // @todo '(ie|oe)'. $plofKlank
+ *
+ *  // http://nl.wikipedia.org/wiki/Medeklinker
+ *  private $plofKlank = '(p|t|k|b|d)';
+ *  private $wrijfKlank = '(f|s|ch|sj|v|z|g|j)';    // journaal
+ *  private $neusKlank = '(m|n|ng)';
+ *  private $vloeiKlank = '(l|r)';
+ *  private $glijKlank = '(j|w)';                   // jaar
+ *  private $affricate = '(ts|zz|tsj|g)';           // /d3/ gin
+ *  private $missingFromWiki = '(c|h|p|q|x|y|z)';
+ *
+ *  public function __construct()
+ *  {
+ *      $this->medeklinker = '(' . $this->missingFromWiki . '|' . $this->plofKlank . '|' . $this->wrijfKlank . '|' . $this->neusKlank . '|' . $this->vloeiKlank . '|' . $this->glijKlank . '|' . $this->affricate . ')';
+ *      $this->medeklinker = '((c|h|p|q|x|y|z)|(p|t|k|b|d)|(f|s|ch|sj|v|z|g|j)|(m|n|ng)|(l|r)|(j|w)|(ts|zz|tsj|g))';
+ *  }
+ */
+class Inflectible
+{
+    /**
+     * @return Transformation[]
```

Comment:
This is incorrect, we are not returning an array here.
```suggestion
     * @return iterable<Transformation>
```